library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/bottomsheet/HomeBottomSheet.kt

```diff
@@ -1,6 +1,7 @@
-package com.epmedu.animeal.home.presentation.ui
+package com.epmedu.animeal.foundation.dialog.bottomsheet
```

Comment:
It is a part of the feeding feature, it shouldn't be in the library. Please move it to feeding.presentation.ui

Reply:
@Fileek  it is not a part of feeding feature. It is a custom bottom sheet dialog, that can be used with custom content anywhere in the app